utils/sessionApiClient.ts

```diff
@@ -0,0 +1,167 @@
+/**
+ * Do not edit this file it is auto-generated by italia-utils / gen-api-models.
```

Comment:
if it is auto-generated it shoulnd't be committed (maybe the comment is outdated?)

Reply:
Not actually. It's generated by an external script (newer version of `io-utils`). I thought it'd be valuable to remind that this is generated code